src/widgets/gauge.rs

```diff
@@ -113,6 +113,11 @@ impl<'a> Widget for Gauge<'a> {
                 buf.get_mut(x, y).set_symbol(" ");
             }
 
+            //set unicode block
+            if self.ratio < 1.0{
+                buf.get_mut(end, y).set_symbol(get_unicode_block(width%1.0));
```

Comment:
I would add an option to use the old low resolution behavior as it is always wanted in environments where those unicode symbols are not well supported.

Reply:
Latest commit adds an option to revert to the old behavior, and I added a test for both new and old behavior